lib/checks/canMultiplyLikeTermConstantNodes.js

```diff
@@ -0,0 +1,30 @@
+const Node = require('../node');
+
+
+// Returns true if the nodes are constant terms with the same constant and has an exponent each
```

Comment:
two comments on your comment (hehe)

1. make sure lines are max 80 char (there are some exceptions, but I like to try to keep it that way)
2. it's often super hard to talk about math nodes clearly (I often write things that are confusing and other ppl have to point it out) but what do you think about "Returns true if `node` is a multiplication of constant nodes where can combine their exponents, e.g. 10^2 * 10^4 * 10 can become 10^7"

wow that was hard for me to rewrite too >.> I think describing it in accurate detail is too long to be worth it, but the example will help and be enough

what do you think?